snapshot_dbg_extension/src/adapter.ts

```diff
@@ -365,12 +372,17 @@ export class SnapshotDebuggerSession extends DebugSession {
         debugLog(response.body);
         this.sendResponse(response);
 
-        if (flushServerBreakpointsToIDE) {
-            flushServerBreakpointsToIDE();
+        if (finalizeInitializePath) {
```

Comment:
Come to think of it, it's also worth tossing a comment here to indicate why we have this finalize step.